K2Bridge.Tests.UnitTests/Visitors/RangeClauseVisitorTests.cs

```diff
@@ -10,102 +10,151 @@ namespace UnitTests.K2Bridge.Visitors
     using global::K2Bridge.Visitors;
     using NUnit.Framework;
 
+    public enum RangeType
+    {
+        Wildcard,
+        Open,
```

Comment:
nit: inclusive, exclusive?

whats the best terms?

Reply:
The docs use inclusive and exclusive https://www.elastic.co/guide/en/elasticsearch/reference/7.15/query-dsl-query-string-query.html#query-string-syntax
So let's go with that